events/klasaReady.js

```diff
@@ -1,14 +1,11 @@
 const { Event } = require("klasa");
-const { PlayerManager } = require("discord.js-lavalink");
+const MusicClient = require("../lib/structures/LavalinkClient");
 
 module.exports = class extends Event {
 
     async run() {
-        // Lavalink
-        this.client.lavalink = new PlayerManager(this.client, this.client.config.nodes, {
-            user: this.client.user.id,
-            shards: this.client.shard.count
-        });
+        // Setup lavalink
+        this.client.lavalink = new MusicClient(this.client, this.client.config.nodes, { shards: this.client.shard.count ? this.client.shard.count : 1 });
```

Comment:
shards is pointless, lol

Reply:
Actually no, for some reason it's unable to get the shard number from the client that's why I added it back.